panic: runtime error: invalid memory address or nil pointer dereference [recovered

soak test snapshot panic
https://jenkins.ops.vega.xyz/blue/organizations/jenkins/common%2Fsnapshot-soak-tests/detail/snapshot-soak-tests/8854/pipeline
```

snapshot general failure restore block: 9218

panic: runtime error: invalid memory address or nil pointer dereference [recovered]

	panic: runtime error: invalid memory address or nil pointer dereference

[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x293b8a2]



goroutine 160 [running]:

code.vegaprotocol.io/vega/cmd/vega/commands/node.(*Command).Run.func1.1()

	/jenkins/workspace/common/system-tests-wrapper/vega/cmd/vega/commands/node/node.go:137 +0x59

panic({0x48f8840, 0x74e3c20})

	/usr/local/go/src/runtime/panic.go:884 +0x212

code.vegaprotocol.io/vega/core/validators.(*Witness).OnTick(0xc001a28090, {0x582f0b8, 0xc00537aea0}, {0x1?, 0x584dce8?, 0x0?})

	/jenkins/workspace/common/system-tests-wrapper/vega/core/validators/witness.go:332 +0x5e2

code.vegaprotocol.io/vega/core/vegatime.(*Svc).notify(...)

	/jenkins/workspace/common/system-tests-wrapper/vega/core/vegatime/service.go:104

code.vegaprotocol.io/vega/core/vegatime.(*Svc).SetTimeNow(0xc001053680, {0x582f0b8, 0xc00537aea0}, {0x40?, 0x470b800?, 0x0?})

	/jenkins/workspace/common/system-tests-wrapper/vega/core/vegatime/service.go:77 +0x24f

code.vegaprotocol.io/vega/core/processor.(*App).OnBeginBlock(_, {{0xc001968380, 0x20, 0x20}, {{0xb, 0x1}, {0xc000118950, 0xb}, 0x2403, {0x34dddb7a, ...}, ...}, ...})

	/jenkins/workspace/common/system-tests-wrapper/vega/core/processor/abci.go:822 +0x747

code.vegaprotocol.io/vega/core/blockchain/abci.(*App).BeginBlock(_, {{0xc001968380, 0x20, 0x20}, {{0xb, 0x1}, {0xc000118950, 0xb}, 0x2403, {0x34dddb7a, ...}, ...}, ...})

	/jenkins/workspace/common/system-tests-wrapper/vega/core/blockchain/abci/abci.go:46 +0x75

code.vegaprotocol.io/vega/cmd/vega/commands/node.(*appW).BeginBlock(_, {{0xc001968380, 0x20, 0x20}, {{0xb, 0x1}, {0xc000118950, 0xb}, 0x2403, {0x34dddb7a, ...}, ...}, ...})

	/jenkins/workspace/common/system-tests-wrapper/vega/cmd/vega/commands/node/app_wrapper.go:64 +0x55

github.com/tendermint/tendermint/abci/client.(*localClient).BeginBlockSync(_, {{0xc001968380, 0x20, 0x20}, {{0xb, 0x1}, {0xc000118950, 0xb}, 0x2403, {0x34dddb7a, ...}, ...}, ...})

	/jenkins/GOPATH/pkg/mod/github.com/vegaprotocol/cometbft@v0.34.28-0.20230322133204-3d8588de736e/abci/client/local_client.go:280 +0x118

github.com/tendermint/tendermint/proxy.(*appConnConsensus).BeginBlockSync(_, {{0xc001968380, 0x20, 0x20}, {{0xb, 0x1}, {0xc000118950, 0xb}, 0x2403, {0x34dddb7a, ...}, ...}, ...})

	/jenkins/GOPATH/pkg/mod/github.com/vegaprotocol/cometbft@v0.34.28-0.20230322133204-3d8588de736e/proxy/app_conn.go:81 +0x55

github.com/tendermint/tendermint/state.execBlockOnProxyApp({0x582d410?, 0xc0000151c0}, {0x58444f0, 0xc001147130}, 0xc00463a5a0, {0x5857218, 0xc00125e060}, 0xc00619cb60?)

	/jenkins/GOPATH/pkg/mod/github.com/vegaprotocol/cometbft@v0.34.28-0.20230322133204-3d8588de736e/state/execution.go:307 +0x3dd

github.com/tendermint/tendermint/state.ExecCommitBlock({0x58444f0, 0xc001147130}, 0xc00463a5a0, {0x582d410, 0xc0000151c0}, {0x5857218?, 0xc00125e060?}, 0xc0000d60c0?)

	/jenkins/GOPATH/pkg/mod/github.com/vegaprotocol/cometbft@v0.34.28-0.20230322133204-3d8588de736e/state/execution.go:537 +0x67

github.com/tendermint/tendermint/consensus.(*Handshaker).replayBlocks(_, {{{0xb, 0x1}, {0xc000efc140, 0x7}}, {0xc000efc150, 0xb}, 0x1, 0x283a, {{0xc000f38940, ...}, ...}, ...}, ...)

	/jenkins/GOPATH/pkg/mod/github.com/vegaprotocol/cometbft@v0.34.28-0.20230322133204-3d8588de736e/consensus/replay.go:471 +0x248

github.com/tendermint/tendermint/consensus.(*Handshaker).ReplayBlocks(_, {{{0xb, 0x1}, {0xc000efc140, 0x7}}, {0xc000efc150, 0xb}, 0x1, 0x283a, {{0xc000f38940, ...}, ...}, ...}, ...)

	/jenkins/GOPATH/pkg/mod/github.com/vegaprotocol/cometbft@v0.34.28-0.20230322133204-3d8588de736e/consensus/replay.go:393 +0x479

github.com/tendermint/tendermint/consensus.(*Handshaker).Handshake(0xc00598ed90, {0x585e320, 0xc000fc9c70})

	/jenkins/GOPATH/pkg/mod/github.com/vegaprotocol/cometbft@v0.34.28-0.20230322133204-3d8588de736e/consensus/replay.go:268 +0x3d4

github.com/tendermint/tendermint/node.doHandshake({_, _}, {{{0xb, 0x1}, {0xc000efc140, 0x7}}, {0xc000efc150, 0xb}, 0x1, 0x283a, ...}, ...)

	/jenkins/GOPATH/pkg/mod/github.com/vegaprotocol/cometbft@v0.34.28-0.20230322133204-3d8588de736e/node/node.go:329 +0x1b8

github.com/tendermint/tendermint/node.NewNode(0xc001135a40, {0x5826dd0, 0xc00015a780}, 0xc001485a90, {0x5809700, 0xc0006cdb78}, 0x0?, 0x0?, 0xc001485ad0, {0x582d410, ...}, ...)

	/jenkins/GOPATH/pkg/mod/github.com/vegaprotocol/cometbft@v0.34.28-0.20230322133204-3d8588de736e/node/node.go:779 +0x597

code.vegaprotocol.io/vega/core/blockchain/abci.NewTmNode({{0x0}, 0x1, 0x1, 0x0, 0x0, {0xc0019862b0, 0xa}, {{0x0}, {0xc0019a00f0, 0x15}}, ...}, ...)

	/jenkins/workspace/common/system-tests-wrapper/vega/core/blockchain/abci/tm_node.go:83 +0x705

code.vegaprotocol.io/vega/cmd/vega/commands/node.(*Command).startABCI(0xc0019b7000, 0x333a73726574656d?, {0x5857c20, 0xc0003b6c80}, {0x7ffc7d87a492, 0x10}, {0x0?, 0x6336323239343633?}, {0x0, 0x0})

	/jenkins/workspace/common/system-tests-wrapper/vega/cmd/vega/commands/node/node.go:412 +0x1b6

code.vegaprotocol.io/vega/cmd/vega/commands/node.(*Command).startBlockchain(0xc0019b7000, 0x3333376134613933?, {0x7ffc7d87a492?, 0x3564386438396231?}, {0x0, 0x0}, {0x0, 0x0})

	/jenkins/workspace/common/system-tests-wrapper/vega/cmd/vega/commands/node/node.go:299 +0x3ab

code.vegaprotocol.io/vega/cmd/vega/commands/node.(*Command).Run.func1()

	/jenkins/workspace/common/system-tests-wrapper/vega/cmd/vega/commands/node/node.go:140 +0x90

created by code.vegaprotocol.io/vega/cmd/vega/commands/node.(*Command).Run

	/jenkins/workspace/common/system-tests-wrapper/vega/cmd/vega/commands/node/node.go:131 +0x66f



percentage of working snapshots: 89.74358974358975% (420/468)

--- a/core/snapshot/engine.go
+++ b/core/snapshot/engine.go
@@ -173,6 +173,7 @@ var nodeOrder = []types.SnapshotNamespace{
 	types.PositionsSnapshot,              // again, needs a market
 	types.SettlementSnapshot,             // needs the market to exist, too
 	types.HoldingAccountTrackerSnapshot,
+	types.EthereumOracleVerifierSnapshot,
 	types.LiquiditySnapshot,
 	types.LiquidityTargetSnapshot,
 	types.StakingSnapshot,

